app/models/sidebar_section.rb

```diff
@@ -26,6 +26,33 @@ class SidebarSection < ActiveRecord::Base
   scope :public_sections, -> { where("public") }
   enum :section_type, { community: 0 }, scopes: false, suffix: true
 
+  def reset_community!
```

Comment:
I think we need more unit tests for this method as the current tests being added does not sufficiently cover everything that this method does.